reposilite-plugins/javadoc-plugin/src/main/kotlin/com/reposilite/plugin/javadoc/JavadocFacade.kt

```diff
@@ -0,0 +1,226 @@
+package com.reposilite.plugin.javadoc
+
+import com.reposilite.journalist.Journalist
+import com.reposilite.journalist.Logger
+import com.reposilite.maven.MavenFacade
+import com.reposilite.maven.api.LookupRequest
+import com.reposilite.plugin.api.Facade
+import com.reposilite.shared.fs.FileType
+import com.reposilite.token.api.AccessToken
+import com.reposilite.web.http.ErrorResponse
+import com.reposilite.web.http.errorResponse
+import io.javalin.http.ContentType
+import io.javalin.http.HttpCode
+import org.intellij.lang.annotations.Language
+import panda.std.Result
+import panda.std.Result.ok
+import java.io.File
+import java.io.FileOutputStream
+import java.nio.file.Files
+import java.nio.file.Path
+
+class JavadocFacade internal constructor(
+    private val javadocFolder: Path,
+    private val mavenFacade: MavenFacade,
+    private val journalist: Journalist
+) : Journalist, Facade {
+
+    private val extractor = DocExtractor()
+
+    fun resolveRequest(repo: String, gav: String, extension: String, accessToken: AccessToken?): Result<JavadocResponse, ErrorResponse> {
+        val target = javadocFolder.resolve("${repo}${File.separator}${gav}")
```

Comment:
`javadocFolder.resolve(repo).resolve(gav)`

Reply:
As before, because currently there is a chance that you'll try to resolve `repo\g/a/v`, because `File.separator` is platform dependent